search-app/docs/api.md

```diff
@@ -787,6 +788,7 @@ __Example__
       "has_variables" : true,
       "has_formats" : false,
       "has_transforms" : false,
+      "has_spatial_subsetting": false
```

Comment:
Needs a comma at the end of the line and probably should put a space before the colon to be consistent with the rest of the fields in the example (and our pretty format apparently has a space).

Reply:
Fixed.